src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -176,30 +177,38 @@ class PerformanceLanding extends React.Component<Props, State> {
       ...location.query,
     };
 
+    const query = decodeScalar(location.query.query) || '';
+    const conditions = tokenizeSearch(query);
+
     // This is a temporary change for trends to test adding a default count to increase relevancy
     if (viewKey === FilterViews.TRENDS) {
       const hasStartAndEnd = newQuery.start && newQuery.end;
       if (!newQuery.statsPeriod && !hasStartAndEnd) {
         newQuery.statsPeriod = DEFAULT_TRENDS_STATS_PERIOD;
       }
-      if (!newQuery.query) {
-        newQuery.query =
-          'count():>1000 transaction.duration:>0 p50():>0 avg(transaction.duration):>0';
+      if (!query) {
+        conditions.setTag('count()', ['>1000']);
+        conditions.setTag('transaction.duration', ['>0']);
       }
-      if (!newQuery.query.includes('count()')) {
-        newQuery.query += (newQuery.query ? ' ' : '') + 'count():>1000';
+      if (!query.includes('count()')) {
+        conditions.setTag('count()', ['>1000']);
       }
-      if (!newQuery.query.includes('transaction.duration')) {
-        newQuery.query += ' transaction.duration:>0';
+      if (!query.includes('transaction.duration')) {
```

Comment:
This has a small edge case where it will false positive on `percentile(transaction.duration, 0.66)` being used as a condition. But that they'll likely hit an error in that situation.

Reply:
I'm going to grab the tags from the tokenizeSearch instead